ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -128,6 +130,7 @@ public void run() {
       initializationException.compareAndSet(null, e);
     }
     if (initializationException.get() == null) {
+      sealedStateChangeCounter.incrementAndGet();
```

Comment:
why do we increment here. ClusterChangeHandler will take care of incrementing and I thought that would suffice